lib/course/middleware/exceptions.rb

```diff
@@ -0,0 +1,26 @@
+module Course
+  module Middleware
+    class Exceptions
+      attr_reader :app
+
+      def initialize(app)
+        @app = app
+      end
+
+      def call(env)
+        app.call(env)
+      # rubocop:disable Lint/RescueException
+      rescue Exception
```

Comment:
you can disable rubocop inline:
`rescue Exception # rubocop:disable Lint/RescueException`
so after that you don't need to enable it.